.env.jungle

```diff
@@ -62,6 +62,7 @@ REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=EOS Jungle Testnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://eosio.online/images/jungle.jpg
 REACT_APP_FOOTER_LINKS=[{"text":"Website","src":"https://jungletestnet.io/"},{"text":"Block Explorer","src":"https://jungle3.bloks.io"},{"text":"Developer Guidelines","src":"https://guide.eoscostarica.io"},{"text":"GitHub","src":"https://github.com/eoscostarica/eosio-dashboard"}]
+REACT_APP_FEATURE_BUG=[{ "text": "Feature Request/Bug Report", "src": "https://github.com/eoscostarica/eosio-dashboard/issues/new/choose"}]
```

Comment:
I think this text reads better like this: 

`"text": "Report a Bug or Request a Feature"`